util/site/fetch_block_stats.py

```diff
@@ -42,11 +42,14 @@ def parse_data_file(rel_path: str) -> Tuple[str, str, str]:
     with (REPO_TOP / rel_path).open() as f:
         block_data = hjson.load(f)
 
+    desc = block_data.get('one_line_desc', '')
```

Comment:
```suggestion
    desc = block_data.get('one_line_desc')
```

Other fields use a json `null` (python `None`) type to encode a field not being present.